pelican/urlwrappers.py

```diff
@@ -122,3 +122,7 @@ def __init__(self, name, *args, **kwargs):
 
 class Author(URLWrapper):
     pass
+
+
+class Index(URLWrapper):
+    pass
```

Comment:
I know, the problem is index is (logically) not an URLWrapper (right now), because of the reason you just described.

You can directly pull the save_as setting, which makes more sense.

If we would revisit how we handle `DIRECT_TEMPLATES` we could add them as URLWrappers, but that is not the issue that you are trying to solve here.